ambry-api/src/main/java/com.github.ambry/config/CloudConfig.java

```diff
@@ -21,6 +21,8 @@
   private final static String VCR_CLUSTER_NAME = "VCRCluster";
   private final static String VCR_CLUSTER_ZK_CONNECT_STRING = "localhost:2181";
   private final static String VCR_REPLICA_MOUNT_PATH_PREFIX = "/tmp/vcr/";
+  public static final String CLOUD_DESTINATION_FACTORY_CLASS = "cloud.destination.factory.class";
+  public static final String DEFAULT_CLOUD_DESTINATION_FACTORY_CLASS = "com.github.ambry.cloud.azure.AzureCloudDestinationFactory";
```

Comment:
You may need a code re-format to meet line length requirement.

Reply:
Fixed.